### PR TITLE
DATACOUCH-650 - Implements deleteById(Iterable<ID> ids).

### PR DESCRIPTION
**Pull request by schauder**
Merge right after DATACMNS-800

### CODE REVIEW

File: src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java [hunk 1]
```diff
@@ -16,15 +16,7 @@
...
-import static org.junit.jupiter.api.Assertions.*;
-
-import java.util.List;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.stream.Collectors;
-
+import com.couchbase.client.core.error.IndexExistsException;
```
Comment (mp911de) [inline src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java:19]: We should verify where import changes stem from. This file seemed to be formatted properly previously.

File: src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java [hunk 2]
```diff
@@ -148,6 +153,25 @@ void count() {
 		}
 	}
 
+	@Test
+		// DATACOUCH-650
```
Comment (mp911de) [inline src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java:157]: Nit: Formatting

**Review by mp911de — CHANGES_REQUESTED** [review]
There are some formatting issues. Other than that, this PR looks fine to me. Paging @daschl and @mikereiche.

**Review by mikereiche — APPROVED** [review]

**Comment by mp911de** [comment]
That's merged and polished now.

**Pull request closed without merging**